domains/transfer/TransferAgreementAutomation/Worker.UnitTests/TransferAgreementsAutomationWorkerTests.cs

```diff
@@ -108,13 +116,29 @@ public async Task WhenCalled_CacheIsSetToHealthy()
             Version = "20231123"
         });
 
-        var worker = new TransferAgreementsAutomationWorker(loggerMock, metricsMock, memoryCache, serviceProviderMock,
-            httpFactoryMock, transferOptions);
+        var worker = new TransferAgreementsAutomationWorker(
+            loggerMock,
+            metricsMock,
+            memoryCache,
+            serviceProviderMock,
+            httpFactoryMock,
+            transferOptions
+            );
 
-        var act = async () => await worker.StartAsync(cts.Token);
-        await act.Invoke();
+        var cts = new CancellationTokenSource();
```

Comment:
Why all this?

Reply:
1. Start the worker with worker.StartAsync(cts.Token)

2. await Task.Delay(1000) to allow the worker to perform some initial operations

3. Then, cancel the worker (cts.Cancel()) and wait for it to acknowledge the cancellation and finish its execution (await workerTask)

Reply:
It did that already with         
var act = async () => await worker.StartAsync(cts.Token);
await act.Invoke();

Reply:
Changed back to original implementation, but with correct Uri, and without using in cancellationtoken: b923fa3